airflow/www_rbac/static/webpack.config.js

```diff
@@ -0,0 +1,128 @@
+/**
+* Licensed to the Apache Software Foundation (ASF) under one
+* or more contributor license agreements. See the NOTICE file
+* distributed with this work for additional information
+* regarding copyright ownership. The ASF licenses this file
+* to you under the Apache License, Version 2.0 (the
+* "License"); you may not use this file except in compliance
+* with the License. You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied. See the License for the
+* specific language governing permissions and limitations
+* under the License.
+*/
+
+const webpack = require('webpack');
+const path = require('path');
+const ManifestPlugin = require('webpack-manifest-plugin');
+const CleanWebpackPlugin = require('clean-webpack-plugin');
+const CopyWebpackPlugin = require('copy-webpack-plugin');
+const ExtractTextPlugin = require('extract-text-webpack-plugin');
+
+// Input Directory (airflow/www_rbac)
+// noinspection JSUnresolvedVariable
+const STATIC_DIR = path.resolve(__dirname, './');
+
+// Output Directory (airflow/www_rbac/static/dist)
+// noinspection JSUnresolvedVariable
+const BUILD_DIR = path.resolve(__dirname, './dist');
+
+const config = {
+  entry: {
+    connectionForm: STATIC_DIR + '/js/connection_form.js',
+    clock: STATIC_DIR + '/js/clock.js',
+    dagreD3: STATIC_DIR + '/js/dagre-d3.js',
+    ganttChartD3v2: STATIC_DIR + '/js/gantt-chart-d3v2.js',
```

Comment:
Ah - this maps into `ganntCharD3v2.$hash.js`. This somewhat breaks the mapping between input filename and output filename - please have these be `'gantt-chart-d3v2': STATIC_DIR + '/js/gantt-chart-d3v2.js'` etc.

Reply:
I agree that it might be confusing for some users, but it has pros and cons. 
To differentiate the local javascript file from the processed javascript file and to eliminate confusion, we usually use different names. 
After every change in javascript file, you'll need to process the javascript file i.e, minimize, format and include dependencies etc. So, if the names in the `<script>` tag in html will be in different format, you'd know that this is a processed file, and you need to make the frontend build again using `npm run build`

Hint: you can simply run `npm run dev` which will constantly watch and automatically makes a build on every javascript change you make in any file.

Reply:
@ashb thoughts on this? ^

Reply:
Not sure - no real strong feelings either way from me.